backend/application/syson-application/src/test/java/org/eclipse/syson/application/controllers/diagrams/general/view/GVEdgePortUsageTests.java

```diff
@@ -193,14 +196,14 @@ public void givenSysMLProjectWithPortsWhenBindingConnectorAsUsageEdgeToolCreatio
                     .extracting(Edge::getStyle, EDGE_STYLE)
                     .hasSourceArrow(ArrowStyle.None)
                     .hasTargetArrow(ArrowStyle.None);
+            bindingConnectorIds.addAll(newEdges.stream().map(Edge::getTargetObjectId).toList());
         };
 
         this.diagramCheckerService.checkDiagram(diagramChecker, this.diagram, this.verifier);
 
-        ISemanticChecker semanticChecker = this.semanticCheckerService.getElementInParentSemanticChecker("part1", SysmlPackage.eINSTANCE.getType_Feature(),
-                SysmlPackage.eINSTANCE.getBindingConnectorAsUsage());
-
-        this.semanticCheckerService.checkEditingContext(semanticChecker, this.verifier);
+        this.semanticCheckerService.checkElement(this.verifier, BindingConnector.class, () -> bindingConnectorIds.get(0), bindingConnector -> {
+            assertThat(bindingConnector.getOwner().getName()).isEqualTo("Package 1");
```

Comment:
I have aligned the computation of the owner with the ConnectionUsage while moving the service to the correct service class. The computation is now coherent.